api/v1alpha1/common.go

```diff
@@ -243,6 +243,28 @@ const (
 	DataProcessType OperationType = "DataProcess"
 )
 
+type AffinityPolicy string
+
+const (
+	DefaultAffinityStrategy AffinityPolicy = ""
+	RequireAffinityStrategy AffinityPolicy = "Require"
+	PreferAffinityStrategy  AffinityPolicy = "Prefer"
+)
+
+type AffinityStrategy struct {
+	// Policy one of: "", "Require", "Prefer"
+	// +optional
+	Policy AffinityPolicy `json:"policy,omitempty"`
+
+	Prefer  []Prefer `json:"prefer,omitempty"`
+	Require []string `json:"require,omitempty"`
+}
+
+type Prefer struct {
```

Comment:
I suggest adding comments for this struct. Thanks.

Reply:
fixed.